app/controllers/api/v1/people_controller.rb

```diff
@@ -74,7 +74,7 @@ def json_api_errors_for(error)
       def import_person_from_nomis prison_number
```

Comment:
tiny point: could we use parentheses in method definitions with parameters, i.e.  `def import_person_from_nomis(prison_number)`

Reply:
yes, I agree